src/components/manifold-oauth/manifold-oauth.tsx

```diff
@@ -36,6 +36,7 @@ export class ManifoldOauth {
         frameborder="0"
         id="manifold-oauth-window"
         name="manifold-oauth-window"
+        sandbox="true"
```

Comment:
I think you want to leave of the `="true"`. According to the documentation, using the attribute without a value applies all restrictions. Then you can peel back restrictions with values like `allow-forms` and `allow-popups`.

Reply:
In fact you might need `allow-scripts`

Reply:
TypeScript threw an error by just setting that attribute without a value. Is there a more specific value we should provide?

Reply:
Hmm! Unrelated but I'm wondering if `allow-same-origin` would allow us to receive a cookie from PUMA!

![image](https://user-images.githubusercontent.com/374078/64282379-eb94af00-cf1a-11e9-8db1-41a48fac9df9.png)